content/en/about/index.md

```diff
@@ -1,7 +1,6 @@
 ---
 date: 2022-10-04T15:01:42Z
 title: "About Us"
-description: "Our mission is to empower modern data teams to build next-generation apps, APIs, and models quickly, reliably, and securely. We help developers and organizations unlock value and growth by getting the right data to the right process at the right time and place. As specialists who occupy the unique intersection of cloud native computing, event-driven data systems, and machine learning, we aim to create shared tools across technical teams that encourage collaboration, empathy, and curiosity."
```

Comment:
This description has been removed and the default one added to the config.toml file will be used.